apps/judicial-system/web/src/shared-components/BlueBox/BlueBox.tsx

```diff
@@ -0,0 +1,9 @@
+import { Box } from '@island.is/island-ui/core'
+import React from 'react'
+import * as styles from './BlueBox.treat'
+
+const BlueBox: React.FC<{}> = (props) => {
+  return <Box className={styles.blueBox}>{props.children}</Box>
+}
```

Comment:
I would like to see box props being used when possible instead of custom styles, this will ensure predefined styles are reuse rather then adding unnecessary styles
```suggestion
const BlueBox: React.FC = ({ children }) => (
  <Box background="blue100" padding={3} borderRadius="large">
    {children}
  </Box>
)
```